deploy/clowdapp.yaml

```diff
@@ -549,6 +549,31 @@ objects:
         WHERE rule_results.result IN ('fail', 'error', 'unknown', 'fixed')
         GROUP BY "rules"."ref_id", "accounts"."org_id";
 
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: compliance-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    queries:
+    - prefix: hms_analytics/compliance/tests
+      chunksize: 200000
+      query: >-
+        SELECT
```

Comment:
Can you also add `ref_id` from the canonical profile? This would enable us to track Compliance policies (stig, cisa, ...).

Reply:
@romanblanco @skateman Can you guys recommend how to get to this data, I see multiple `ref_id`'s in the schema and have no idea which is the right one?

Reply:
@shaded-enmity, to get `ref_id` of the canonical profile[1], `canonical_profiles`[2] need to be joined on `tailorings.profile_id`[3].

[1] https://github.com/RedHatInsights/compliance-backend/blob/fc9709a63c9e1c2d21b03799d87ea076e60ab886/db/schema.rb#L286
[2] https://github.com/RedHatInsights/compliance-backend/blob/fc9709a63c9e1c2d21b03799d87ea076e60ab886/db/schema.rb#L284
[3] https://github.com/RedHatInsights/compliance-backend/blob/fc9709a63c9e1c2d21b03799d87ea076e60ab886/db/schema.rb#L311

Reply:
Thanks @romanblanco, added.